src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -461,16 +480,42 @@
     }
   }
 
+  .topRowWrapper {
+    display: flex;
+    flex-direction: row;
```

Comment:
this is also not needed, row is the default anyway

Reply:
Fixed